orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -33,8 +33,15 @@ class OWCorpusViewer(OWWidget):
     inputs = [(IO.DATA, Table, 'set_data')]
     outputs = [(IO.MATCHED, Corpus, widget.Default), (IO.UNMATCHED, Corpus)]
 
-    search_indices = ContextSetting([0])   # features included in search
-    display_indices = ContextSetting([0])  # features for display
+    settingsHandler = PerfectDomainContextHandler(
+        match_values = PerfectDomainContextHandler.MATCH_VALUES_ALL
+    )
+
+    list_search_indices = ContextSetting([], exclude_metas=False)   # features included in search
```

Comment:
Why not keep the names `search_indices` and `display_indices`?

If you would really like to have `list` in the name as well, the I suggest to refactor to `search_list_indices` and `display_list_indices` for easier readability.